Whats new november (#21788)
dotnet/docs: docs/whats-new/2020-11.md

---
title: ".NET docs: What's new for November 1, 2020 - November 30, 2020"
description: "What's new in the .NET docs for November 1, 2020 - November 30, 2020."
ms.date: 12/02/2020
---

> # .NET docs: What's new for November 1, 2020 - November 30, 2020

Reviewer: ```suggestion
# .NET docs: What's new for November 2020
```

> Welcome to what's new in the .NET docs from November 1, 2020 through November 30, 2020. This article lists some of the major changes to docs during this period.

Reviewer: ```suggestion
Welcome to what's new in the .NET docs for November 2020. This article lists some of the major changes to docs during this period.
```

> ## .NET

Reviewer: Why do we differentiate .NET and .NET fundamentals?

Author: We probably shouldn't.  What are your thoughts @tdykstra ?

Reviewer 2: I agree, we shouldn't differentiate. The division is based on standard vs. core TOC groupings, which are now combined in fundamentals.

Author: Done. I combined them by hand for this month. We'll make a code change to fix it for next month.

### New articles

- [Runtime libraries overview](../standard/runtime-libraries-overview.md) - Reorg Fundamentals TOC
- [Best practices for displaying and persisting formatted data](../standard/base-types/best-practices-display-data.md) - Behavior changes due to NLS -> ICU switch on Windows
- [Behavior changes when comparing strings on .NET 5+](../standard/base-types/string-comparison-net-5-plus.md) - Behavior changes due to NLS -> ICU switch on Windows
- [Extract elements from a string](../standard/base-types/parse-strings.md) - Move alternatives out of `String.Split` reference

### Updated articles

- [Compare strings in .NET](../standard/base-types/comparing.md) - Behavior changes due to NLS -> ICU switch on Windows
- [Standard date and time format strings](../standard/base-types/standard-date-and-time-format-strings.md) - Q2 Content performance updates

> - [Serialization guidelines](../standard/serialization/serialization-guidelines.md) - .NET Framework -> .NET (/standard/s*)

Reviewer: ```suggestion
- [Serialization guidelines](../standard/serialization/serialization-guidelines.md) - .NET Framework -> .NET
```

- [.NET runtime method events](../fundamentals/diagnostics/runtime-method-events.md) - Document CoreCLR runtime provider events
- [.NET runtime thread pool events](../fundamentals/diagnostics/runtime-thread-events.md) - Document CoreCLR runtime provider events
- [.NET runtime type events](../fundamentals/diagnostics/runtime-type-events.md) - Document CoreCLR runtime provider events

## .NET IoT libraries

### New articles

- [Debug .NET apps on Raspberry Pi](../iot/debugging.md) - .NET IoT landing page
- [Deploy .NET apps to Raspberry Pi](../iot/deployment.md) - .NET IoT landing page
- [.NET IoT Libraries documentation](../iot/index.yml) - .NET IoT landing page
- [Develop apps for IoT devices with the .NET IoT Libraries](../iot/intro.md) - .NET IoT landing page
- [Quickstart - Use .NET to drive a Raspberry Pi Sense HAT](../iot/quickstarts/sensehat.md) - .NET IoT landing page
- [Read values from an analog-to-digital converter](../iot/tutorials/adc.md) - .NET IoT landing page
- [Blink an LED](../iot/tutorials/blink-led.md) - .NET IoT landing page
- [Display text on an LCD](../iot/tutorials/lcd-display.md) - .NET IoT landing page

> - [Read environmental conditions from a sensor](../iot/tutorials/temp-sensor.md) - .NET IoT landing page
> 
> ## C# language

Reviewer 3: ```suggestion
- [Read environmental conditions from a sensor](../iot/tutorials/temp-sensor.md) - .NET IoT landing page

## .NET Desktop

### New articles

- [What's new (Windows Forms .NET)](/dotnet/desktop/winforms/whats-new/?view=netdesktop-5.0) - Add what's new in WinForms article

### Updated articles

- [How to migrate a Windows Forms desktop app to .NET 5](/dotnet/desktop/winforms/migration/?view=netdesktop-5.0) - Add Visual Basic notes to WinForms migration article
- [Desktop guide](/dotnet/desktop/?view=netdesktop-5.0) - Fix multiple related to open issues.

## C# language
```

## F# language

### New articles

- [Interpolated strings](../fsharp/language-reference/interpolated-strings.md) - Add nameof and interpolated strings articles for F#
- [Nameof](../fsharp/language-reference/nameof.md) - Add nameof and interpolated strings articles for F#
- [FS0025: Incomplete pattern match](../fsharp/language-reference/compiler-messages/fs0025.md) - FS0025
- [What's new in F# 5.0](../fsharp/whats-new/fsharp-50.md) - What's new in F# 5.0

### Updated articles

- [Interactive programming with F\#](../fsharp/tutorials/fsharp-interactive/index.md) - Update F# Interactive reference for F# 5

## Community contributors

The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).

- [amadosoft](https://github.com/amadosoft) - Dado (12)
- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (8)
- [nschonni](https://github.com/nschonni) - Nick Schonning (7)
- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (6)
- [hyoshioka0128](https://github.com/hyoshioka0128) - Hiroshi Yoshioka (4)

> - [azure-sdk](https://github.com/azure-sdk) - Azure SDK Bot (3)

Reviewer: ```suggestion
```

- [wuyuansushen](https://github.com/wuyuansushen) (3)
- [nojaf](https://github.com/nojaf) - Florian Verdonck (2)
- [AngelMunoz](https://github.com/AngelMunoz) - Angel D. Munoz (1)
- [benmccallum](https://github.com/benmccallum) - Ben McCallum (1)
- [brettrowberry](https://github.com/brettrowberry) (1)
- [btamsevi](https://github.com/btamsevi) (1)
- [c0g1t8](https://github.com/c0g1t8) - Gary Chan (1)
- [chrisa0](https://github.com/chrisa0) (1)
- [chrisnas](https://github.com/chrisnas) (1)
- [DMiradakis](https://github.com/DMiradakis) - Daniel Miradakis (1)
- [duracellko](https://github.com/duracellko) - Rastislav Novotný (1)
- [DXPetti](https://github.com/DXPetti) (1)
- [ebresafegaga](https://github.com/ebresafegaga) - Oghenevwogaga Ebresafe (1)
- [ebwinters](https://github.com/ebwinters) - Ethan Winters (1)
- [ericmutta](https://github.com/ericmutta) - Eric Mutta (1)
- [facundo91](https://github.com/facundo91) - Facundo Ezequiel Aita (1)
- [FLGMwt](https://github.com/FLGMwt) - Ryan Stelly (1)
- [hede](https://github.com/hede) (1)
- [hunanniu](https://github.com/hunanniu) - Bo Zhang (1)
- [jeremyVignelles](https://github.com/jeremyVignelles) - Jérémy VIGNELLES (1)

> - [juho-hanhimaki](https://github.com/juho-hanhimaki) - Juho Hanhimäki (1)

Reviewer 3: Add user
```suggestion
- [joneskj55](https://github.com/joneskj55) - Kevin Jones (1)
```

- [kinow](https://github.com/kinow) - Bruno P. Kinoshita (1)
- [maa](https://github.com/maa) - macart (1)
- [mamachanko](https://github.com/mamachanko) - Max Brauer (1)

> - [mfcallahan](https://github.com/mfcallahan) - Matt Callahan (1)

Reviewer 3: Add user
```suggestion
- [mfcallahan](https://github.com/mfcallahan) - Matt Callahan (1)
- [MHDeveloper30](https://github.com/MHDeveloper30) - MHDeveloper30 (1)
```
